### ISSUE TITLE
How to create textured vertex

### ISSUE THREAD

**Issue by edwinsyarief** [faiface/pixel#276]
Hi @faiface, is it possible to render line with texture? I want to create rope with texture.

Thank you.

**Comment by jakubDoka** [comment]
you have two options, or you use repeating texture on multiple sprites and chain them together, assuming you already have phizycs for your rope, or you go deep into pixel source and learn how sprite works. 

Breafly every vertex (called TriangleData in pixel) has texture position intensity and color, you care about texture, i recomend you to study some opengl, as what you are demanding is low level technical issue. I coudl do this for you but have no time rn and dont want to advertize my products here.

To sum it up pisxel is small lib that does as match with as little. You can easily use it as framework to create new functionality, all you have to do is not limit your self only to sprite and batch but dig deep into it.
